Lucidtech/Las/ApiClient.cs

```diff
@@ -45,10 +44,13 @@ public class ApiClient : Client
         /// <returns>
         /// Prediction on document
         /// </returns>
-        public Prediction Predict(string documentPath, string modelName, string consentId)
+        public Prediction Predict(string documentPath, string modelName, string consentId = "default")
         {
             string contentType = GetContentType(documentPath);
-            string documentId = UploadDocument(documentPath, contentType, consentId);
+            byte[] body = File.ReadAllBytes(documentPath);
+            var postDocumentsResponse = JsonSerialPublisher.ObjectToDict<Dictionary<string, object>>(
+                PostDocuments(body, contentType, consentId));
+            string documentId = (string)postDocumentsResponse["documentId"];
```

Comment:
If `documentId` key is not present in the response, this line will throw an exception.

Reply:
This seems to be the case in the python sdk as well, should we agree on a way to handle this consistently? @augustak

Reply:
documentId will always be present